demo/src/javascripts/actions/app-actions.js

```diff
@@ -1,35 +1,39 @@
 import {request, text} from 'd3-request';
+
 import {StreamParser} from '../utils/worker-utils';
 
-export function loadContent(filename) {
+const loadContentSuccess = (name, content) => {
```

Comment:
I personally find using `function` keyword clearer than arrow syntax for plain function declaration.  But I guess it is the trend in redux apps to use arrow notation for everything, so the change might be appropriate here in the `demo` folder. Up to @Pessimistress to decide.